src/Extensions/azure-storage/azure-storage/Microsoft.Diagnostics.Monitoring.AzureStorage.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>$(ToolTargetFrameworks)</TargetFrameworks>
+		<RuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</RuntimeIdentifiers>
+		<PackAsToolShimRuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</PackAsToolShimRuntimeIdentifiers>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<RootNamespace>Microsoft.Diagnostics.Monitoring.AzureStorage</RootNamespace>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>disable</Nullable>
```

Comment:
This isn't enabled by default, so you shouldn't have to disable it. If left explicitly disabled, when we go to enable it across the board, this may fall under the radar.